webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -32,11 +31,12 @@ const DefaultEmptyState = (props) => {
   return (
     <EmptyStatePattern
       icon={icon}
+      iconType={iconType || 'pf'}
       header={header}
       description={description}
       documentation={documentation ? documentationBlock(documentation) : null}
-      action={<PrimaryActionButton action={action} />}
```

Comment:
The fact that I added PrimaryActionButton and SecondaryActionButtons here is because that's the default action buttons that are used in Foreman.

However, in Katello the action buttons (both Primary and Secondary) are using Links that wrap a button.
Thus, I introduced `EmptyStatePattern` that allow any plugin to use it in order to create their own Empty State component.